pdm/cli/commands/config.py

```diff
@@ -43,10 +43,13 @@ def _list_config(self, project: Project, options: argparse.Namespace) -> None:
         ui.echo("Home configuration ({}):".format(project.global_config._config_file))
         with ui.indent("  "):
             for key in sorted(project.global_config):
+                try:
+                    global_config_item = project.global_config._config_map[key]
```

Comment:
Since the key is already excluded in `__iter__` so this shouldn't raise KeyError anymore. But keeping this is fine and  I will merge it.
Thanks